CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 * **New Data Source:** `session-application-session-policy datasource` ([#94](https://github.com/pingidentity/terraform-provider-pingfederate/pull/94))
 * **New Data Source:** `protocol-metadata-lifetime-settings datasource` ([#100](https://github.com/pingidentity/terraform-provider-pingfederate/pull/100))
 * **New Data Source:** `server-settings-general-settings datasource` ([#101](https://github.com/pingidentity/terraform-provider-pingfederate/pull/101))
+* **New Data Source:** `server-settings-log-settings datasource` ([#104](https://github.com/pingidentity/terraform-provider-pingfederate/pull/104))
```

Comment:
I missed this in the other reviews, but these should use underscores, not hyphens